client/python/lib/README.md

```diff
@@ -13,10 +13,18 @@ Then get to the client library directory:
 
 `cd model_server/client/python/lib`
 
-Make sure you have `setuptools` installed in your Python environment:
+Make sure you have `setuptools`, `protobuf` and `protobuf-compiler` installed in your Python environment:
```

Comment:
Please split pip packages from apt ones. Also, the protobuf package does not need to be installed here manually as it has been added to `setup.py`